src/src/AdminBundle/Controller/EventController.php

```diff
@@ -151,11 +205,37 @@ public function assignOrganizerAction($id, $event_id, Request $request)
         return $this->redirectToRoute('events_edit', ['id' => $id, 'event_id' => $event_id]);
     }
 
+    /**
+     * @Route("event_type/edit/{id}/event/{event_id}/edit/assign/language",
+     *     name="events_assign_language",
+     *     requirements={"id"="\d+", "event_id"="\d+"}
+     * )
+     * @Method("POST")
+     */
+    public function assignLanguageAction($id, $event_id, Request $request)
+    {
+        $em = $this->getDoctrine()->getManager();
+        $language = $em->getRepository(Language::class)->findOneBy(
+            ['language' => $request->get(self::$modalInputLanguages)]
+        );
+
+        $event = $em->getRepository(Event::class)->findOneBy(['id' => $event_id]);
+
+        $eventLanguage = new EventLanguages();
```

Comment:
Co sa stane ked takyto `EventLanguages` uz assignuty je? Btw, daj jednotne cislo pre klasu.

Reply:
Vsimni si v navrhu, tam sa tabulka vola `event_languages`. Preto mi prislo konzistentne nzvat entitu `EventLanguages`. Keby to mam teraz menit, je vela miest, kde by som to menila. Naposledy ked som nieco rename-ovala cez editor, skoncilo to velmi zle.